internal/processor/title.go

```diff
@@ -32,39 +29,31 @@ func processTitle(title string, matchRelease bool) []string {
 		t.Add(replace, matchRelease)
 	}
 
-	if strings.ContainsAny(title, "-") {
-		strip := strings.ReplaceAll(title, "-", "?")
-		replace := strings.ReplaceAll(strip, " ", "?")
-		t.Add(replace, matchRelease)
-	}
-
-	if strings.ContainsAny(title, "!") {
-		strip := strings.ReplaceAll(title, "!", "?")
-		replace := strings.ReplaceAll(strip, " ", "?")
-		t.Add(replace, matchRelease)
-	}
-
-	if strings.ContainsAny(title, ":") {
-		strip := strings.ReplaceAll(title, ":", "")
-		replace := strings.ReplaceAll(strip, " ", "?")
-		t.Add(replace, matchRelease)
-
-		split := strings.Split(title, ":")
-		if len(split) > 1 {
-			first := strings.ReplaceAll(split[0], " ", "?")
-			second := strings.ReplaceAll(strings.Trim(split[1], " "), " ", "?")
-			part := fmt.Sprintf("%v*%v", first, second)
-
-			t.Add(part, matchRelease)
+	if re, err := regexp.Compile(`\s\(([A-Za-z]{2})\)$`); err == nil {
```

Comment:
from here to line 58 could probably be replaced by this, but would need further testing:
```
if regexp.MustCompile(`[[:punct:]]`).MatchString(title) {
		regexQuestionmark := regexp.MustCompile(`[?]{2,}`)
		regexRemove := regexp.MustCompile(`[^[:alnum:]]`)

		title = strings.TrimRight(title, ".!")
		title = strings.ReplaceAll(title, "'", "")
		title = strings.ReplaceAll(title, "’", "")
		title = strings.ReplaceAll(title, ",", "")
		title = strings.ReplaceAll(title, "&", "and")

		replace := regexRemove.ReplaceAllString(title, "?")
		replace = regexQuestionmark.ReplaceAllString(replace, "*")
		replace = strings.TrimRight(replace, "?*")
		t.Add(replace, matchRelease)
	}
```